Source/Extensions/ConstraintsSetup.swift

```diff
@@ -55,14 +66,92 @@ extension BottomContainerView {
     addConstraint(NSLayoutConstraint(item: topSeparator, attribute: .Height,
       relatedBy: .Equal, toItem: nil, attribute: .NotAnAttribute,
       multiplier: 1, constant: 1))
+
+  }
+
+  func setupCompactHeightConstraints() {
+    for attribute: NSLayoutAttribute in [.Height, .Left, .Top] {
+      addConstraint(NSLayoutConstraint(item: topSeparator, attribute: attribute,
+        relatedBy: .Equal, toItem: self, attribute: attribute,
+        multiplier: 1, constant: 0))
+    }
+
+    addConstraint(NSLayoutConstraint(item: doneButton, attribute: .CenterX,
```

Comment:
I think a better way is to define 2 sets of constraints, and toggle their `active` accordingly